core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1974,7 +1974,7 @@ public function edit($intId=null, $ajaxId=null)
 
 					$this->strField = $vv;
 					$this->strInputName = $vv;
-					$this->varValue = $currentRecord[$vv] ?? null;
```

Comment:
Because there was no null check before and you did not add one in the `editAll()` method (line 2424).

<img width="493" alt="" src="https://user-images.githubusercontent.com/1192057/176130567-5fb32b41-180d-488b-80ea-e06c206f1c85.png">

We might also keep the null check here and add it in the `editAll()` method instead.